watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Tasks/TaskDetailFragment.java

```diff
@@ -114,19 +123,22 @@ public void onClick(View v) {
     }
 
     @Override
+    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
+        menu.clear();
+        inflater.inflate(R.menu.delete_menu, menu);
+//        super.onCreateOptionsMenu(menu, inflater);
```

Comment:
Don't we still need to call this (or at least in the case that we don't handle the click)?